script/functions.sh

```diff
@@ -11,12 +11,6 @@ export JRUBY_OPTS=${JRUBY_OPTS:-"--server -Xcompile.invokedynamic=false"}
 SPECS_HAVE_RUN_FILE=specs.out
 MAINTENANCE_BRANCH=`cat maintenance-branch`
 
-# Don't allow rubygems to pollute what's loaded. Also, things boot faster
-# without the extra load time of rubygems. Only works on MRI Ruby 1.9+
-if is_mri_192_plus; then
-  export RUBYOPT="--disable=gem"
-fi
-
```

Comment:
Good to see that you're giving this a second thought: https://github.com/rspec/rspec-dev/commit/e0639d522d952f883b6c4deca929231630afa775